girderformindlogger/api/v1/user.py

```diff
@@ -378,7 +378,7 @@ def updateIDCode(self, id, code):
     @access.public(scope=TokenScope.USER_INFO_READ)
     @autoDescribeRoute(
         Description('Remove an ID Code from a user.')
-        .param('id', 'Profile ID', required=True)
+        .modelParam('id', model=UserModel, level=AccessType.ADMIN)
```

Comment:
[`ProfileModel`](https://github.com/henryrossiter/mindlogger-app-backend/blob/fb15cb391e0b6c632388c53305e91931cdbba802/girderformindlogger/api/v1/user.py#L18) not `UserModel`. And `AccessType.ADMIN` is probably too strict.